src/components/QuestionnaireForm/QuestionnaireForm.jsx

```diff
@@ -711,6 +765,8 @@ export default class QuestionnaireForm extends Component {
       reference:
         this.getPatient()
     };
+    this.addAuthorToResponse(qr, this.getPractitioner());
+    console.log(qr);
```

Comment:
Can you add some message before logging qr to help people see the extension is added  from the console log without going through the debugger? 
```suggestion
    console.log("GetQuestionnaireResponse final QuestionnaireResponse: ", qr);
```